swagger_to_sdk_config.json

```diff
@@ -6,18 +6,18 @@
       "dep ensure",
       "go install",
       "cd ../..",
-      "go generate",
+      "go generate ./profiles/...",
       "gofmt -w ./services/",
       "gofmt -w ./profiles/"
     ],
     "autorest_options": {
-      "use": "@microsoft.azure/autorest.go@preview",
+      "use": "@microsoft.azure/autorest.go@2.1.86",
```

Comment:
There is a npm syntax to say "latest of the 2.x series" if you want.
https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-

Because Autorest gives that to npm directly, and doesn't look at what you actually write.

Reply:
Updated! Thanks for the tip

Reply:
For posterity, I tested the syntax to use with this command:
``` bash
npm show "@microsoft.azure/autorest.go@~2" version
```